torch/csrc/jit/runtime/static/generated_ops.cpp

```diff
@@ -5191,10 +5191,12 @@ REGISTER_NATIVE_OPERATOR_FUNCTOR(
           const auto& self = p_node->Input(0).toTensor();
           const auto cuda_enabled = p_node->Input(1).toBool();
           const auto cpu_enabled = p_node->Input(2).toBool();
-          const auto cuda_dtype = p_node->Input(3).toScalarType();
-          const auto cpu_dtype = p_node->Input(4).toScalarType();
+          const auto xla_enabled = p_node->Input(3).toBool();
```

Comment:
@tenpercent can you take a look at this? should we just leave static runtime out?